ivy/functional/backends/numpy/creation.py

```diff
@@ -214,6 +214,9 @@ def zeros(
     return _to_device(np.zeros(shape, dtype), device=device)
 
 
+zeros.support_native_out = True
```

Comment:
What was your reasoning for adding this line?

Reply:
I had added this line as this is a numpy primary function and it supports out argument. I referred the Inplace Updates section of the Deep Dive.